sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -1409,4 +1409,17 @@ class DynamicPartitionPruningSuiteAEOff extends DynamicPartitionPruningSuiteBase
   with DisableAdaptiveExecutionSuite
 
 class DynamicPartitionPruningSuiteAEOn extends DynamicPartitionPruningSuiteBase
-  with EnableAdaptiveExecutionSuite
+  with EnableAdaptiveExecutionSuite {
+  test("simple inner join triggers DPP with mock-up tables test") {
```

Comment:
What does this test?

Reply:
Only for debug. I will remove this test later.